content/en/docs/tasks/administer-cluster/change-default-storage-class.md

```diff
@@ -70,7 +70,7 @@ for details about addon manager and how to disable individual addons.
 
 1. Mark a StorageClass as default:
 
-      Similarly to the previous step, you need to add/set the annotation
+      Similar to the previous step, you need to add/set the annotation
```

Comment:
I'm not sure what is most idiomatic English. I prefer “similarly”. A possible alternative:
```suggestion
      In a similar way to the previous step, you need to add/set the annotation
```

Reply:
Hi @gprasath .
You could remove the clause ("Similar to the previous step") and clarify the sentence before the code block.


To mark a StorageClass as default, [you need to] set the annotation `storageclass.kubernetes.io/is-default-class=true`, for example: